infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
+STAGE = os.environ.get("STAGE")
+GH_HOOK_SECRET = os.environ.get("GH_HOOK_SECRET")
+
+sns = boto3.client("sns")
+
+
+class SNSTopicNotFoundError(Exception):
+    """Raised when trying to publish to a non-existent SNS topic"""
+    pass
+
+
+# e.g. "arn:aws:sns:eu-west-1:1234:dev_gh_team_events"
+def _topic_arn(gh_event):
+    return "{}:{}_gh_{}_events".format(
+        SNS_ARN,
+        STAGE,
+        gh_event,
+    )
+
+
+def _publish(topic_arn, message):
+    print("Publishing to '{}'. Message: '{}'".format(
+        topic_arn,
+        message
+    ))
+
+    try:
+        sns.publish(
+            TopicArn=topic_arn,
+            Message=message
+        )
+    except ClientError as e:
+        if e.response["Error"]["Code"] == "NotFound":
```

Comment:
Boto3 error handling is somehow low level, that's why I wrapped it in a custom `SNSTopicNotFoundError` exception, to simplify the client code a little bit.